chef/cookbooks/keystone/templates/default/openrc.erb

```diff
@@ -2,6 +2,12 @@
 export OS_USERNAME='<%= @keystone_settings['admin_user'] %>'
 export OS_PASSWORD='<%= @keystone_settings['admin_password'] %>'
 export OS_TENANT_NAME='<%= @keystone_settings['default_tenant'] %>'
+export OS_IDENTITY_API_VERSION='<%= @keystone_settings['api_version'] %>'
+<% if @keystone_settings['api_version'] != '2.0' %>
+export OS_USER_DOMAIN_NAME='Default'
+export OS_PROJECT_DOMAIN_NAME='Default'
+export OS_AUTH_VERSION='<%= @keystone_settings['api_version'] %>'
```

Comment:
is this intentionally inside the <% if> block? the OS_IDENTITY_API_VERSION is outside